packages/manager/modules/nasha/src/index.js

```diff
@@ -3,47 +3,69 @@ import '@ovh-ux/manager-core';
 import '@uirouter/angularjs';
 import 'oclazyload';
 
+import '@ovh-ux/ng-ui-router-breadcrumb';
+
 const moduleName = 'ovhManagerNashaLazyLoading';
 
 angular
-  .module(moduleName, ['ui.router', 'oc.lazyLoad', 'ovhManagerCore'])
+  .module(moduleName, [
+    'ui.router',
+    'oc.lazyLoad',
+    'ovhManagerCore',
+    'ngUiRouterBreadcrumb',
+  ])
   .config(
-    /* @ngInject */ ($stateProvider) => {
+    /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
       const lazyLoad = ($transition$) => {
         const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
 
-        return import('./nasha.module').then((mod) =>
+        return import('./dashboard/nasha.module').then((mod) =>
           $ocLazyLoad.inject(mod.default || mod),
         );
       };
 
       $stateProvider
-        .state('nasha.**', {
-          url: '/paas/nasha/:nashaId',
-          lazyLoad,
+        .state('nasha', {
+          url: '/nasha',
+          redirectTo: 'nasha.index',
+          template: '<div ui-view></div>',
+          resolve: {
+            breadcrumb: () => 'NASHA',
+          },
         })
-        .state('nashas.**', {
-          url: '/paas/nasha',
+        .state('nasha.index.**', {
+          url: '',
           lazyLoad: ($transition$) => {
             const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
 
-            return import('./nashas/nasha.module').then((mod) =>
+            return import('./nasha.module').then((mod) =>
               $ocLazyLoad.inject(mod.default || mod),
             );
           },
         })
-        .state('nasha-order-complete.**', {
-          url: '/nasha/order/complete',
+        .state('nasha.dashboard.**', {
+          url: '/:nashaId',
           lazyLoad,
         })
-        .state('nasha-add.**', {
-          url: '/nasha/new',
+        .state('nasha.nasha-order-complete.**', {
+          url: '/order/complete',
           lazyLoad,
         })
-        .state('nasha-unavailable.**', {
-          url: '/nasha/unavailable',
+        .state('nasha.nasha-add.**', {
+          url: '/new',
+          lazyLoad,
+        })
+        .state('nasha.nasha-unavailable.**', {
+          url: '/unavailable',
           lazyLoad,
         });
+
+      $urlRouterProvider.when(/^\/paas\/nasha/, () => {
```

Comment:
Is it possible to use `redirectTo` instead ?

Reply:
@varun257 A `redirectTo` will only work for root url (if customer target `/paas/nasha`). If I target `/paas/nasha/{serviceName}` the `redirectTo` won't apply except if for each nasha state we create a paas.nasha state that adds a `redirectTo`, it is easier to maintain the redirection that way imo